engine.js

```diff
@@ -242,7 +243,8 @@ var hmdajson = require('./lib/hmdajson'),
      * -----------------------------------------------------
      */
 
-    HMDAEngine.fileToJson = function(file, spec, next) {
+    HMDAEngine.fileToJson = function(file, year, next) {
+        var spec = hmdaRuleSpec.getFileSpec(year);
```

Comment:
Already does..
If you look at `getFileSpec` itself: https://github.com/cfpb/hmda-pilot/blob/master/app/index.html#L35  you'll notice that when it can't get the `spec` it returns an empty object. When that empty object gets passed into `hmdajson.process()` it does a validation on the `spec` passed in and returns an `err` as appropriate.